### PR TITLE
Fix Context Export Resource Messages

### PR DESCRIPTION
**Pull request by kingthorin**
Tweak resource keys in Messages.properties and resource usage in ContextExportDialog to actually refer to export not import as appropriate.

### CODE REVIEW

File: src/lang/Messages.properties [hunk 1]
```diff
@@ -894,7 +895,6 @@ context.import.label.context	   = Context:
 context.import.label.dir		   = Directory:
 context.import.label.file		   = File:
 context.import.label.overwrite	   = Overwrite:
```
Comment (thc202) [inline src/lang/Messages.properties]: Worth fixing these as well? (Note that in the export dialogue the other resource keys, labels and error messages, are also using import.)

Comment (kingthorin) [inline src/lang/Messages.properties]: Sure, I'll take a look at it.

Comment (thc202) [inline src/lang/Messages.properties]: Thank you.

**Comment by kingthorin** [comment]
:zap:

**Comment by thc202** [comment]
Thank you!

**Pull request merged**